src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32Program.cs

```diff
@@ -294,7 +294,7 @@ public List<ContextMenuResult> ContextMenus(string queryArguments, IPublicAPI ap
                     AcceleratorModifiers = ModifierKeys.Control | ModifierKeys.Shift,
                     Action = _ =>
                     {
-                        Main.StartProcess(Process.Start, new ProcessStartInfo("explorer", ParentDirectory));
+                        Wox.Infrastructure.Helper.OpenInShell(ParentDirectory);
```

Comment:
Any reason why in this case `Wox.Infrastructure.` is used?

Reply:
Conflict with `public static IShellLinkHelper Helper { get; set; } = new ShellLinkHelper();`
I will fix this